src/steam_network/protocol/protobuf_client.py

```diff
@@ -74,91 +148,167 @@ async def wait_closed(self):
         pass
 
     async def run(self):
+        jobs_to_process = 0
         while True:
-            for job in self.job_list.copy():
-                logger.info(f"New job on list {job}")
-                if job['job_name'] == "import_game_stats":
-                    await self._import_game_stats(job['game_id'])
-                    self.job_list.remove(job)
-                elif job['job_name'] == "import_collections":
-                    await self._import_collections()
-                    self.job_list.remove(job)
-                elif job['job_name'] == "import_game_times":
-                    await self._import_game_time()
-                    self.job_list.remove(job)
-                else:
-                    logger.warning(f'Unknown job {job}')
+            if jobs_to_process < 2:
+                for job in self.job_list[:1].copy():
```

Comment:
This code significantly improves the overall load time because jobs used to throttle the code because we couldn't receive anything until it finished. Realistically, it's a band-aid, a proper fix would be not to use the job list (legacy code) and properly await sending off these jobs so it's nonblocking. 

Cleaning this up is part of a larger rewrite that I'm in the process of but it won't be ready for some time.

Reply:
swapped it to [:2] so the list actually does something